GVFS/GVFS.Installer.Mac/CreateMacInstaller.sh

```diff
@@ -110,10 +114,17 @@ function CopyBinariesToInstall()
 
 function CreateInstaller()
 {
-    pkgBuildCommand="/usr/bin/pkgbuild --identifier $INSTALLERPACKAGEID --scripts \"${SCRIPTSPATH}\" --root \"${STAGINGDIR}\" \"${BUILDOUTPUTDIR}\"$INSTALLERPACKAGENAME.pkg"
+    pkgBuildCommand="/usr/bin/pkgbuild --identifier $INSTALLERPACKAGEID --scripts \"${SCRIPTSPATH}\" --root \"${STAGINGDIR}\" \"${PACKAGESTAGINGDIR}/$INSTALLERPACKAGENAME.pkg\""
     eval $pkgBuildCommand || exit 1
 }
 
+function CreateGVFSDistribution()
+{
+    buildGVFSDistCmd="/usr/bin/productbuild --distribution \"$SCRIPTSPATH/Distribution_GVFS.xml\" --package-path \"$PACKAGESTAGINGDIR\" \"${BUILDOUTPUTDIR}$INSTALLERPACKAGENAME.pkg\""
```

Comment:
Another spot: `buildGVFSDistCmd` -> `buildVFSForGitDistCmd`

(Mentioning these because from what I've seen we're using "VFSForGit" naming scheme in this file)